### PR TITLE
CUDA: 9.1

### PR DESCRIPTION
**Pull request by ax3l**
Add the latest CUDA release, 9.1

### CODE REVIEW

File: var/spack/repos/builtin/packages/cuda/package.py [hunk 1]
```diff
@@ -50,7 +52,7 @@ class Cuda(Package):
             url="http://developer.download.nvidia.com/compute/cuda/6_5/rel/installers/cuda_6.5.14_linux_64.run")
 
     def install(self, spec, prefix):
-        runfile = glob(join_path(self.stage.path, 'cuda*run'))[0]
```
Comment (davydden) [inline var/spack/repos/builtin/packages/cuda/package.py]: was it a bug or shall this be version dependent?

**Comment by ax3l** [comment]
They changed th file name -.-

File: var/spack/repos/builtin/packages/cuda/package.py (hunk 1 above)
Comment (davydden) [reply var/spack/repos/builtin/packages/cuda/package.py]: But then this line must be conditional on the version

**Comment by ax3l** [comment]
It should still work, it's just an other regex for the file name :)

File: var/spack/repos/builtin/packages/cuda/package.py (hunk 1 above)
Comment (ax3l) [reply var/spack/repos/builtin/packages/cuda/package.py]: don't worry, the modified regex fits both old releases and the latest one that changed the file name :)

**Pull request merged**